packages/axios-extension/src/abap/adt-catalog/handlers/transport.ts

```diff
@@ -100,11 +101,34 @@ function getTransportableList(doc: Document): string[] {
  * @param doc
  * @returns
  */
-function getLockedTransport(doc: Document): string[] {
-    const transportNum = xpath.select1('//LOCKS//REQ_HEADER/TRKORR/text()', doc)?.toString();
-    if (transportNum) {
-        return [transportNum];
+function getLockedTransport(doc: Document): TransportRequest[] {
+    const transportReqEle = xpath.select1('//LOCKS//REQ_HEADER', doc) as Element;
```

Comment:
Do we have a test covering this case?

Reply:
Yes

Reply:
Sorry, my question was not very clear. Is there a test for the case when the transport is locked? I could find a test for when it is not locked. If I'm mistaken, would you mind pasting a permalink to the test which covers this

Reply:
@ullasholla 

Locked for existing project: 

https://github.com/SAP/open-ux-tools/blob/e47ce0a8aad8c990e0098b103f7a5409dc215fb5/packages/axios-extension/test/abap/abap-service-provider.test.ts#L187